org.gnome.gThumb.yml

```
@@ -0,0 +1,81 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+
+command: gthumb
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-public-share
+  - --filesystem=xdg-pictures
+
```

```suggestion
  - --filesystem=xdg-run/gvfs
  - --filesystem=xdg-run/gvfsd
  - --talk-name=org.gtk.vfs.*
```

This will provide gvfs access and display the mounts.

---

Yes, I did try it before opening the PR. It lets you see the mounts but you cannot open it. Did you try it?

org.gnome.Photos has the same problem.